docs/security/how-to-use-claimmapper.md

```diff
@@ -0,0 +1,103 @@
+---
+id: how-to-use-claimmapper
+title: How to Use ClaimMapper
+sidebar_label: How to Use ClaimMapper
+---
+
+`ClaimMapper` is a plugin that extracts claims from JSON Web Tokens (JWT). This process is achieved with the method `GetClaims`, which translates `AuthInfo` structs from the caller into `Claims` about the caller's roles within Temporal.
+
+A `Role` (within Temporal) is a bit mask that combines one or more of the role constants.
+
+```go
+role := authorization.RoleReader | authorization.RoleWriter
+```
+
+`GetClaims` is customizable, and can be modified with the `temporal.WithClaimMapper` server option. Temporal also offers a default JWT `ClaimMapper` for your use.
+
+A typical approach is for `ClaimMapper` to interpret custom `Claims` from a caller's JWT access token, such as membership in groups, and map them to Temporal roles for the user. The subject information from the caller's TLS certificate can also be a parameter in determining roles.
+
+### `AuthInfo`
+
+`AuthInfo` is a struct that is passed to `GetClaims`. `AuthInfo` contains an authorization token extracted from the `authorization` header of the gRPC request.
+
+`AuthInfo` includes a pointer to the `pkix.Name` struct. This struct contains an x.509 distinguishable name from the caller's mTLS certificate.
```

Comment:
I had to look up what an X.509 DN was. [IBM](https://www.ibm.com/docs/en/ibm-mq/7.5?topic=certificates-distinguished-names) of all places has a good page explaining what they are... and surprisingly nowhere else does. I was going to suggest finding and picking a page to link outbound to for information, but maybe everyone just links to IBM.

Reply:
I can link to IBM. That's fine